.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
you can rename the script file, to have a meaningful name, e.g. `run-codespell.sh`

Reply:
Done